unicodetools/data/emoji/dev/emoji-data.txt

```diff
@@ -1,11 +1,11 @@
 # emoji-data.txt
-# Date: 2022-05-06, 16:14:52 GMT
+# Date: 2022-05-13, 21:54:24 GMT
 # © 2022 Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see https://www.unicode.org/terms_of_use.html
 #
 # Emoji Data for UTS #51
-# Version: 15.0
+# Used with Emoji Version 15.0 and subsequent minor revisions (if any)
```

Comment:
This is good, but please keep the versions of these files that are in the real place in sync with these ones:
https://github.com/unicode-org/unicodetools/tree/main/unicodetools/data/ucd/dev/emoji

... manually until the relevant issue is fixed ...

(That is, please copy both files to ...data/ucd/dev/emoji and include those files in the same PR.)